tools/simulator/frameworks/runtime-src/proj.ios_mac/mac/SimulatorApp.mm

```diff
@@ -446,19 +446,18 @@ - (void) startup
     NSString *resourcePath = [[NSBundle mainBundle] resourcePath];
     FileUtils::getInstance()->addSearchPath(resourcePath.UTF8String);
 
-    // app
-    _app = new AppDelegate();
-
-    [self setupUI];
-    [self adjustEditMenuIndex];
+    // not show the custom menu items
+    // [self setupUI];
+    // [self adjustEditMenuIndex];
```

Comment:
custom menu items needed native feature disappear at native runtime. such as, set window scale, display fps or not.

So I comment the `setupUI` code, now the simulator works well without this feature

Reply:
I think these features are needed, and should be implemented in JS later.